tests/h/search/query_test.py

```diff
@@ -3,8 +3,10 @@
 
 import pytest
 import webob
+import mock
```

Comment:
Nitpick until we're auto-formatting code with Black - groups of imports should generally be sorted.